projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.stats.toolbelt.quickslot": "快捷栏",
+    "tetra.stats.toolbelt.quickslot.tooltip": "有 §e%s§r 个快速插槽",
+    "tetra.stats.toolbelt.potion_storage": "药水栏",
+    "tetra.stats.toolbelt.potion_storage.tooltip": "有 §e%s§r 个药水空间",
+    "tetra.stats.toolbelt.storage": "储物栏",
+    "tetra.stats.toolbelt.storage.tooltip": "有 §e%s§r 个储物空间",
+    "tetra.stats.toolbelt.quiver": "箭袋栏",
+    "tetra.stats.toolbelt.quiver.tooltip": "有 §e%s§r 个箭袋空间",
+    "tetra.stats.toolbelt.booster": "力量提升",
+    "tetra.stats.toolbelt.booster.tooltip": "提供跳跃提升和强度为 §e%s§r 的有限制的飞行",
+    "tetra.stats.bow.releaseLatch": "释放销",
+    "tetra.stats.bow.releaseLatch.tooltip": "在弓被拉满时自动射箭",
+    "tetra.stats.bow.overbowed": "重弓",
+    "tetra.stats.bow.overbowed.tooltip": "这把弓拉起来费力。弓只能被持续拉满 §7§e%s§r 秒",
+    "tetra.stats.bow.overbowed.tooltip_extended": "超出这个上限时弓会被松开并且不射出弓箭。弓会进行 §70.5§r 秒的冷却",
+    "tetra.stats.multishot": "多重射击",
+    "tetra.stats.multishot.tooltip": "每一发射出多个弹射物。\n \n§7数量：§e%s\n§7散布：§e%s",
+    "tetra.stats.zoom": "放大",
+    "tetra.stats.zoom.tooltip": "当瞄准或按住 [shift] 时放大 §e%sx§r 倍",
+    "tetra.stats.velocity": "速度",
+    "tetra.stats.velocity.tooltip": "增加射出投射物的初速度 §e%s （%s）",
+    "tetra.stats.suspend": "悬浮",
+    "tetra.stats.suspend.tooltip": "当从拉满的弓射出时，投射物的初速度增加 §e100%%§r 并无视重力",
+    "tetra.stats.criticalStrike": "暴击",
+    "tetra.stats.criticalStrike.tooltip": "当攻击或破坏方块时有几率暴击.\n \n§e%s§7 的概率造成 §e%s §7伤害\n§e%s§7 的概率立即破坏方块\n \n§7对于异常坚硬的方块不会产生效果",
+    "tetra.stats.holo_speed": "冷却",
+    "tetra.stats.holo_speed.tooltip": "使用特殊功能有 §e%sx§r 倍的冷却时间",
+    "tetra.stats.holo.scannerRange": "搜寻半径",
+    "tetra.stats.holo.scannerRange.tooltip": "扫描器可以触及最远 §e%s§r 个方块的地方",
+    "tetra.stats.holo.scannerHorizontalSpread": "水平搜寻范围",
+    "tetra.stats.holo.scannerHorizontalSpread.tooltip": "扫描器有 §e%s 个单位的水平搜索范围\n§7单位夹角：§e%s 度",
+    "tetra.stats.holo.scannerVerticalSpread": "垂直搜寻范围",
+    "tetra.stats.holo.scannerVerticalSpread.tooltip": "扫描器覆盖了 §e%s 度的垂直范围",
+    "tetra.stats.holo.scannerVerticalSpread.tooltip_extended": "垂直范围被分成三部分，中间一部分覆盖了围绕中心的 §e20 度§7。上部和下部覆盖了 §e+-25 度§7以上的一切。上部和下部只在垂直范围大于等于 §e50 度§7时有效",
+    "tetra.stats.intuit": "直觉",
+    "tetra.stats.intuit.tooltip": "杀死目标或破坏方块所获得的经验以 §e%sx§r 的比例使打磨值增加",
+    "tetra.stats.earthbind": "地缚",
+    "tetra.stats.earthbind.tooltip": "攻击生物时有 §e10%%§r 的概率将它锁在原地 §e%s 秒§r。更容易出现在更低的高度，概率在基岩处达到最大的 §e50%%§r。",
+    "tetra.stats.magicCapacity": "魔力容量",
+    "tetra.stats.magicCapacity.tooltip": "组件可以容纳的魔力量，可以通过附魔之类的升级来使用。\n \n§7魔力容量：§e%s",
+    "tetra.stats.stability": "稳定性",
+    "tetra.stats.stability.tooltip": "将这个物品主要组件的有效魔力容量改变 §e%s 点",
+    "tetra.stats.stabilizing": "稳定化",
+    "tetra.stats.stabilizing.tooltip_short": "增加这个物品主要组件的有效魔力容量",
+    "tetra.stats.unstable": "不稳定",
+    "tetra.stats.unstable.tooltip_short": "减少这个物品主要组件的有效魔力容量",
+    "tetra.stats.workable": "易加工",
+    "tetra.stats.workable.tooltip": "打磨这个物品的耗时减少 §e%s§r 次",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
    "tetra.stats.workable.tooltip": "打磨这个物品的需要次数减少 §e%s§r",
```
```suggestion
    "tetra.stats.workable.tooltip": "打磨这个物品的耗时减少 §e%s§r",
```

我没玩过这个模组，需要从这两个当中选一个。